src/cli.ts

```diff
@@ -26,7 +26,7 @@ export interface ClientInfo {
 
 export class ValidationError extends Error {
 	public constructor(
-		type: "not-found" | "version",
+		public type: "not-found" | "version",
```

Comment:
If we're going to do this, perhaps we should also export a type of the possible types like:

```ts
export type ValidationErrorType = "not-found" | "version";
```

And use that type here, and export it from `src/index.ts`.

Reply:
I like it!